enterprise/frontend/src/embedding-sdk/components/public/NewQuestion/NewQuestion.tsx

```diff
@@ -0,0 +1,117 @@
+import { useState } from "react";
+import { usePreviousDistinct } from "react-use";
+
+import { withPublicComponentWrapper } from "embedding-sdk/components/private/PublicComponentWrapper";
+import {
+  Notebook,
+  QuestionVisualization,
+} from "embedding-sdk/components/public/InteractiveQuestion/components";
+import {
+  InteractiveQuestionProvider,
+  useInteractiveQuestionContext,
+} from "embedding-sdk/components/public/InteractiveQuestion/context";
+import {
+  SaveQuestionForm,
+  SaveQuestionTitle,
+} from "metabase/components/SaveQuestionForm";
+import { SaveQuestionProvider } from "metabase/components/SaveQuestionForm/context";
+import { useCreateQuestion } from "metabase/query_builder/containers/use-create-question";
+import { useSaveQuestion } from "metabase/query_builder/containers/use-save-question";
+import { Button, Group, Stack, Tabs, Title, Box } from "metabase/ui";
+
+const SaveQuestion = ({ onClose }: { onClose: () => void }) => {
+  const { question, originalQuestion } = useInteractiveQuestionContext();
+
+  const handleCreate = useCreateQuestion();
+  const handleSave = useSaveQuestion();
+
+  if (!question) {
+    return null;
+  }
+
+  return (
+    <SaveQuestionProvider
+      question={question}
+      originalQuestion={originalQuestion ?? null}
+      onCreate={handleCreate}
+      onSave={handleSave}
+      multiStep={false}
+      initialCollectionId={null}
+    >
+      <Stack p="md">
+        <Title>
+          <SaveQuestionTitle />
+        </Title>
+        <SaveQuestionForm onCancel={onClose} />
+      </Stack>
+    </SaveQuestionProvider>
+  );
+};
+
+const ResetQuestionButton = () => {
+  const { onReset } = useInteractiveQuestionContext();
+
+  return <Button onClick={onReset}>Reset</Button>;
+};
+
+const NewQuestionInner = () => {
+  const [activeTab, setActiveTab] = useState<
+    "notebook" | "visualization" | "save" | null | (string & unknown)
+  >("notebook");
+
+  const previousTab = usePreviousDistinct(activeTab);
+
+  const returnToPreviousTab = () => {
+    if (previousTab) {
+      return setActiveTab(previousTab);
+    }
+  };
+
+  const onSaveButtonClick = () => {
```

Comment:
just a UX thing, in this screen there are 2 save buttons
![image](https://github.com/user-attachments/assets/d093db8e-0e97-4fc2-8dff-858d08aad992)

And reset and cancel sound pretty similar, but they do different thing as well.

it feels as if when you show cancel + save buttons combo, you should hide reset + save buttons at the top.

What do you think?

Reply:
yeah, we can hide them

Reply:
done